Security/Helper/Nonce.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace HWI\Bundle\OAuthBundle\Security\Helper;
+
+final class Nonce
+{
+    /**
+     * @return string
+     */
+    public static function generate()
+    {
+        return md5(microtime(true).uniqid('', true));
```

Comment:
spaces around: ``` . ```

Reply:
Not according to the CS fixer, unfortunately.

Reply:
seems like fixer fixed too much - many not related files were changed

Reply:
I think that's just because the cs fixer was added later and people ignored it, lol.

Reply:
in that case it is better to have a separate PR to update it because we are not sure how long will it take to merge this one

Reply:
I can always drop the commit if people don't want me to add it.